api/base/utils.py

```diff
@@ -56,3 +55,18 @@ def waterbutler_url_for(request_type, provider, path, node_id, token, obj_args=N
 
     url.args.update(query)
     return url.url
+
+
+def custom_exception_handler(exc, context):
```

Comment:
Let's go ahead and move this into a new api/base/exceptions.py

Also, the name is a bit generic. Perhaps jsonapi_exception_handler()?